src/grpcd/runtime.rs

```diff
@@ -287,18 +289,22 @@ impl Farcaster for FarcasterService {
             }))
             .await?;
         match oneshot_rx.await {
-            Ok(BusMsg::Info(InfoMsg::SwapInfo(info))) => {
+            Ok(BusMsg::Info(InfoMsg::SwapInfo(SwapInfo {
+                swap_id: _,
+                connection,
+                connected,
+                state: _,
```

Comment:
Instead of discarding individual variable use `..` and remove swap id and state

Reply:
This is done on purpose. It reminds the programmer to update the GRPC interface in case the incoming object changes.